pyramid_oereb/core/hook_methods.py

```diff
@@ -124,16 +125,52 @@ def produce_sld_content(params, real_estate_config):
     return template.render(**template_params)
 
 
+def compare(a, b):
+    """
+    This is the method to sort a plr list (while respecting the theme order).
+    This standard sorts the plrs within themes, you can set your configuration
+    to a different method if you need a specific sorting.
+
+    Args:
```

Comment:
Maybe, for Args the exact types could be listed. I would also reformulate the sentence for Returns.

Reply:
Done.